app/models/compute_resource.rb

```diff
@@ -132,7 +132,7 @@ def host_compute_attrs(host)
 
   def host_interfaces_attrs(host)
     host.interfaces.select(&:physical?).each.with_index.reduce({}) do |hash, (nic, index)|
-      hash.merge(index.to_s => nic.compute_attributes)
+      hash.merge(index.to_s => nic)
```

Comment:
I don't think changing this is a good idea, it'll completely change the per-NIC attributes passed to every CR's interface creation, breaking them all. It might be best to _merge_ the general NIC model attributes (`ip`, `ip6`) with the compute_attributes.